src/overlays/actors/ovl_Item_Ocarina/z_item_ocarina.c

```diff
@@ -15,7 +15,17 @@ void ItemOcarina_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void ItemOcarina_Update(Actor* thisx, GlobalContext* globalCtx);
 void ItemOcarina_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void ItemOcarina_SetupAction(ItemOcarina* this, ItemOcarinaActionFunc actionFunc);
```

Comment:
Functions should usually only be forward declared when necessary, which isn't the case for at least a few of these.
```suggestion
```